db/schema.rb

```diff
@@ -175,11 +176,11 @@
     t.boolean  "send_planting_reminder",  default: true
   end
 
-  add_index "members", ["confirmation_token"], name: "index_members_on_confirmation_token", unique: true, using: :btree
-  add_index "members", ["email"], name: "index_members_on_email", unique: true, using: :btree
-  add_index "members", ["reset_password_token"], name: "index_members_on_reset_password_token", unique: true, using: :btree
-  add_index "members", ["slug"], name: "index_members_on_slug", unique: true, using: :btree
-  add_index "members", ["unlock_token"], name: "index_members_on_unlock_token", unique: true, using: :btree
+  add_index "members", ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true, using: :btree
```

Comment:
Again with the "user" thing. We don't use that terminology. What is the intent of this change?